soc/arm/atmel_sam0/common/soc_samr34.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021 Argentum Systems Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/gpio.h>
+#include <soc.h>
+
+#if !(ATMEL_SAM0_DT_SERCOM_CHECK(4, atmel_sam0_spi) && CONFIG_SPI_SAM0)
+/* When the radio is not in use, it's important that #CS is set high, to avoid
+ * unexpected behavior and increased current consumption... see Chapter 10 of
+ * DS70005356C. We also hold the radio in reset.
+ */
+static int soc_pinconf_init(const struct device *dev)
+{
+	const struct device *portb = DEVICE_DT_GET(DT_NODELABEL(portb));
+
+	ARG_UNUSED(dev);
+
+	if (!device_is_ready(portb)) {
+		return -ENXIO;
+	}
+
+	gpio_pin_configure(portb, 31, GPIO_OUTPUT_HIGH);
+	gpio_pin_configure(portb, 15, GPIO_OUTPUT_LOW);
+
+	return 0;
+}
+
```

Comment:
why aren't these pins described in DT?

Reply:
These pins are set to hold the radio silicon in reset when the SPI peripheral is _not_ in use... is there a way to set GPIO pins in the DT, and if so, can we set them for this negated state?

Reply:
One can do things like https://github.com/zephyrproject-rtos/zephyr/pull/43817 to avoid hardcoded numbers in code.

Reply:
(note: non-blocking to get this in, though)